catalogs/catalog-lakehouse-paimon/src/main/java/org/apache/gravitino/catalog/lakehouse/paimon/GravitinoPaimonTable.java

```diff
@@ -58,20 +70,26 @@ protected TableOperations newOps() {
    * @return The converted Paimon table.
    */
   public Schema toPaimonTableSchema() {
-    Schema.Builder builder = Schema.newBuilder().comment(comment).options(properties);
-    if (partitioning != null) {
-      builder.partitionKeys(
-          Arrays.stream(partitioning)
-              .map(
-                  partition -> {
-                    NamedReference[] references = partition.references();
-                    Preconditions.checkArgument(
-                        references.length == 1,
-                        "Partitioning column must be single-column, like 'a'.");
-                    return references[0].toString();
-                  })
-              .collect(Collectors.toList()));
+    Schema.Builder builder = Schema.newBuilder().comment(comment);
+    if (properties == null) {
+      properties = Maps.newHashMap();
+    }
+    if (partitioning == null) {
+      partitioning = EMPTY_PARTITIONING;
     }
+    if (indexes == null) {
```

Comment:
we had checked the nullability of indexs already, this seems redundant?

Reply:
`toPaimonTableSchema` is called in unit test, afraid that passing `null` will cause `NPE`.

Reply:
I prefer to change the test code to remove the check

Reply:
removed this check.